Lib/test/test_venv.py

```diff
@@ -632,6 +637,17 @@ def test_activate_shell_script_has_no_dos_newlines(self):
                 error_message = f"CR LF found in line {i}"
                 self.assertFalse(line.endswith(b'\r\n'), error_message)
 
+    @requireVenvCreate
+    def test_create_git_ignore_file(self):
+        """
+        Test that a .gitignore file is created.
+        The file should contain a `*\n` line.
+        """
+        self.run_with_capture(venv.create, self.env_dir, scm_ignore_file='git')
+        file_lines = self.get_text_file_contents('.gitignore').splitlines()
+        self.assertIn('*', file_lines)
```

Comment:
Should there be here another test that checks that when the user uses `--without-scm-ignore-file` there's no such file created? 🤔 Would be nice to get coverage reports here, so we can automatically catch such missing test cases 🤔

Reply:
Yep, and I already realized it was missing, but I was waiting until I decided whether I was going to give you your iterable interface or not. 😉